backend/src/test/java/gov/cdc/usds/simplereport/api/TestResultTest.java

```diff
@@ -55,4 +57,33 @@ private ArrayNode fetchTestResults(ObjectNode variables) {
         return (ArrayNode) runQuery("test-result-query", variables).get("testResults");
     }
 
+    // TODO: THIS DOESN'T WORK
+    // @Test
+    // public void submitTestResult() throws Exception {
+    //     Person p = _dataFactory.createFullPerson(_org);
+    //     _dataFactory.createTestEvent(p, _site);
+
+    //     System.out.printf("person: %s", p.toString());
+    //     String dateTested = "2020-12-31T14:30:30.000Z";
+    //     // SimpleDateFormat sdf = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss.SSS'Z'");
+  
+    //     // TODO: add real variables here
+    //     ObjectNode variables = JsonNodeFactory.instance.objectNode()
+    //         .put("deviceId", "ff86e743-5876-4117-9936-2932fbdbcec2")
```

Comment:
this test fails, presumably from this line. Not sure how to easily add a mock device id without a device factory function